blockprotocol/blockprotocol
# Fix React hydration issues related to # in paths

[Asana task](https://app.asana.com/0/1200211978612931/1202736533402964/f) (internal links)

The goal of this PR is to fix two Sentry issues (internal links):

- [There was an error while hydrating](https://sentry.io/organizations/hashintel/issues/3476303382/?project=6603021)
- [Hydration failed because the initial UI does not match what was rendered on the server.](https://sentry.io/organizations/hashintel/issues/3476303370/?project=6603021)

Related upstream issue:
- https://github.com/vercel/next.js/issues/25202

To test:
1.  Open docs page
1.  Open dev tools
1.  Navigate to a header and see `#[id]` appended to the URL
1.  Refresh the page
1.  Ensure no errors in console
1.  Repeat for mobile layout (narrow document width)
